LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2359,6 +2380,8 @@ private static void GetIteConfigurationsC(ISuperIO superIO, Manufacturer manufac
                     case Model.Z790_AORUS_PRO_X: // ITE IT87952E
                     case Model.Z690_AORUS_PRO:
                     case Model.Z690_AORUS_MASTER: // ITE IT87952E
+                    case Model.X870_AORUS_ELITE_WIFI7: // ITE IT87952E
+                    case Model.X870E_AORUS_ELITE_WIFI7: // ITE IT87952E
                         v.Add(new Voltage("Vcore", 0));
```

Comment:
On the X870E model, Fan 4 is referenced as a potential pump header. 5/6 aren’t included. I would avoid including X870/X870E on Intel 600/700 entries as they typically vary drastically.

Reply:
I removed the X870E model from the PR because it seems to differ quite a bit from the non E. I also moved the X870 to it's own case.